src/codewhisperer/models/constants.ts

```diff
@@ -241,6 +241,6 @@ export const accessTokenMigrationDoNotShowAgain = `Don\'t Show Again`
 export const accessTokenMigrationDoNotShowAgainKey = 'CODEWHISPERER_ACCESS_TOKEN_MIGRATION_DO_NOT_SHOW_AGAIN'
 
 export const accessTokenMigrationDoNotShowAgainLastShown =
-    'CODEWHISPERER_ACCESS_TOKEN_MIGRATION_DO_NOT_SHOW_AGAIN_LAST_SHOWN'
+    'CODEWHISPERER_ACCESS_TOKEN_MIGRATION_DO_NOT_SHOW_AGAIN_LAST_SHOWN_TIME'
```

Comment:
Future note: please avoid verbose names like this. This is 2x longer than any name should be.

Reply:
noted will be sure to keep shorter in the future